ddtrace/ext/tags.go

```diff
@@ -9,8 +9,15 @@ package ext
 
 const (
 	// TargetHost sets the target host address.
+	// Deprecated: Use NetworkDestinationName instead for hostname and NetworkDestinationIP for IP addresses
 	TargetHost = "out.host"
 
+	// NetworkDestinationName is the remote hostname or similar where the outbound connection is being made to.
+	NetworkDestinationName = "network.destination.name"
+
+	// NetworkDestinationIP is the remote address where the outbound connection is being made to.
+	NetworkDestinationIP = "network.destination.ip"
```

Comment:
it looks like this isn't used anywhere?

Reply:
It will be used in the future for Unified Naming Conventions of tags, but I added it now so people who see the other deprecated tags understand that hostname/ip cant be combined anymore but rather separated into different tags